pyonepassword/op_items/_new_field_registry.py

```diff
@@ -16,8 +18,89 @@ def __init__(self, msg, item_dict=None):
         self.item_dict = item_dict
 
 
+class OPNewItemField(OPItemField):
+    """
+    A class for creating new fields for use with new items
+
+    New fields may be created directly, or from existing fields
+    """
+    FIELD_TYPE = Optional[None]
```

Comment:
`Optional[T]` is a shortcut for `Union[T, None]` so what you've written here is actually `Union[None, None]`

Reply:
derp...typo. supposed to be `FIELD_TYPE: Optional[str] = None`

good catch, thanks